manifests/m/Mondago/GoIntergrator/3.2.1.18541.22072/Mondago.GoIntergrator.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Mondago.GoIntergrator
+PackageVersion: 3.2.1.18541.22072

```

This will have to match Control Panel if there is a way to perform a silent install
```suggestion
PackageVersion: 3.2.1.18541
```